superset-frontend/src/cccs-viz/plugins/plugin-chart-application-links/src/ApplicationLinks.tsx

```diff
@@ -1,53 +1,104 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { ApplicationsProps } from './types';
 import styles from './styles';
 
 export default function ApplicationLinks(props: ApplicationsProps) {
   const { application, appVal, appType } = props;
-  const [alfredCount] = useState(-1);
+  const [alfredCount, setAlfredCount] = useState(-1);
+  const [alfredURL, setAlfredURL] = useState(
+    'https://alfred-pb.chimera.cyber.gc.ca',
```

Comment:
This hard-coded URL is just the default value, right? If the default value should always be overwritten, should we perhaps put in a more obviously default value here? Just so we'll fail loudly if the default value is ever not overwritten on PB?

Reply:
That is correct, it is the default value, do you know of a good default value to put here?

Reply:
It could probably be something dumb like 'default-alfred-url'? I think? Maybe?

Reply:
Updated the default URL to be `default-alfred-url` in the latest commit